### DISCUSSION TITLE
Ability to not play a sound on /msg despite global setting

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#7485] [Enhancement]
As per Discord user `@Tasty_Goat🐐#4962`;

The request to add a prefix maybe, or extend the -s perhaps, to mute the sound for `/cmi msg`, just like how `!` does. So despite the global config.yml setting to play audible, this would suppress it.

**Comment by MrCrazys** [comment]
> As per Discord user `@Tasty_Goat🐐#4962`;
> 
> The request to add a prefix maybe, or extend the -s perhaps, to mute the sound for `/cmi msg`, just like how `!` does. So despite the global config.yml setting to play audible, this would suppress it.

`!` doesn't mute the sound, it simply sends a clean message without the prefix. I don't really see a need for the `-s` bit. I just feel like it would be too hard to keep track of, especially if a player messages `-s` or you want to message `-s`. I feel like the easiest way is to turn off the sound. But I do see why it could be a good idea.
